lib/block.go

```diff
@@ -25,7 +25,7 @@ type Block struct {
 	Hash      string `json:"hash"`
 	Confirmed string `json:"confirmed"`
 	Proposer  string `json:"proposer"` /* Node.Address() */
-	Round     Round  `json:"round"`
+	PrevRound Round  `json:"round"`
```

Comment:
@Charleslee522 `PrevRound` is correct term? The `Round` information of `Block` is created and used for creating that `Block`. I think, the previous term `Round` is more proper.